core/src/main/java/org/apache/carbondata/core/indexstore/BlockletDataMapIndexStore.java

```diff
@@ -257,10 +264,22 @@ public void put(TableBlockIndexUniqueIdentifierWrapper tableBlockIndexUniqueIden
         for (BlockDataMap blockletDataMap : dataMaps) {
           blockletDataMap.convertToUnsafeDMStore();
         }
+        // get cacheExpirationTime for table from tableProperties
+        String cacheExpirationTime = null;
+        if (null != tableBlockIndexUniqueIdentifierWrapper.getCarbonTable().getTableInfo()
```

Comment:
When it can be null ? I think null validation is not required.

Reply:
removed